include/exadg/time_integration/time_int_abm_base.h

```diff
@@ -0,0 +1,266 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2023 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_TIME_INTEGRATION_TIME_INT_ABM_BASE_H_
+#define INCLUDE_EXADG_TIME_INTEGRATION_TIME_INT_ABM_BASE_H_
+
+
+#include <exadg/time_integration/ab_constants.h>
+#include <exadg/time_integration/am_constants.h>
+#include <exadg/time_integration/push_back_vectors.h>
+#include <exadg/time_integration/time_int_multistep_base.h>
+
+namespace ExaDG
+{
+template<typename Operator, typename VectorType>
+class TimeIntABMBase : public TimeIntMultistepBase
```

Comment:
I tend to write `ABM` explicitly as `AdamsBashforthMoulton` in the class name.

In my opinion, a comment / some documentation for the class should explain that this is an explicit time integrator of predictor-corrector type. One possible interpretation of ABM could be that this class unites AB time integrators and AM time integrators, and potentially also the predictor-corrector version ABM.

Reply:
[d16cfd3](https://github.com/exadg/exadg/pull/585/commits/d16cfd35392b2acbf7f018d2db2649329285102b)